### PR TITLE
update several statements

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3289
- Other reference link(s):

### CODE REVIEW

File: sql-statements/sql-statement-commit.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Query OK, 0 rows affected (0.01 sec)
 
 * In MySQL, with the exception of Group Replication with multiple primaries, it is not typical that a `COMMIT` statement could result in an error. By contrast, TiDB uses optimistic concurrency control and conflicts may result in `COMMIT` returning an error.
 * Be default, `UNIQUE` and `PRIMARY KEY` constraint checks are deffered until statement commit. This behavior can be changed by setting `tidb_constraint_check_in_place=TRUE`.
+* TiDB only has syntactic support for `CompletionTypeWithinTransaction`. Closing the connection or continuing to open a new transaction commit option is not supported.
```
Comment (ran-huang) [inline sql-statements/sql-statement-commit.md:40]:
```suggestion
* TiDB only has syntactic support for `CompletionTypeWithinTransaction`. Closing the connection or continuing to open a new transaction after the transaction is committed is not supported.
```

File: sql-statements/sql-statement-rollback.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Empty set (0.01 sec)
 
 ## MySQL compatibility
 
-This statement is understood to be fully compatible with MySQL. Any compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
+This statement is partly compatible with MySQL. TiDB only has syntactic support for `CompletionTypeWithinTransaction`. Closing the connection or continuing to open a new transaction commit option is not supported. Any compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
```
Comment (ran-huang) [inline sql-statements/sql-statement-rollback.md:39]:
```suggestion
This statement is partly compatible with MySQL. TiDB only has syntactic support for `CompletionTypeWithinTransaction`. Closing the connection or continuing to open a new transaction after the transaction is rolled back is not supported. Any compatibility differences should be [reported via an issue](/report-issue.md) on GitHub.
```

File: sql-statements/sql-statement-set-transaction.md [hunk 3]
```diff
@@ -61,7 +61,8 @@ mysql> SHOW SESSION VARIABLES like 'transaction_isolation';
 
 * TiDB supports the ability to set a transaction as read-only in syntax only.
 * The isolation levels `READ-UNCOMMITTED` and `SERIALIZABLE` are not supported.
-* The isolation level `REPEATABLE-READ` is technically Snapshot Isolation. The name `REPEATABLE-READ` is used for compatibility with MySQL.
+* Snapshot isolation (Snapshot Isolation) technology to achieve optimistic transaction `REPEATABLE-READ` isolation level, partly compatible with MySQL.
```
Comment (ran-huang) [inline sql-statements/sql-statement-set-transaction.md:64]:
```suggestion
* The `REPEATABLE-READ` isolation level is achieved through using the snapshot isolation technology, which is compatible with MySQL.
```

File: sql-statements/sql-statement-start-transaction.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ aliases: ['/docs/dev/reference/sql/statements/start-transaction/']
 
 # START TRANSACTION
 
-This statement starts a new transaction inside of TiDB. It is similar to the statements `BEGIN` and `SET autocommit=0`.
+This statement starts a new transaction inside of TiDB. It is similar to the statements `BEGIN`.
```
Comment (ran-huang) [inline sql-statements/sql-statement-start-transaction.md:10]:
```suggestion
This statement starts a new transaction inside of TiDB. It is similar to the statement `BEGIN`.
```

**Comment by jackysp** [comment]
PTAL @ran-huang

File: sql-statements/sql-statement-commit.md [hunk 5]
```diff
@@ -36,8 +36,8 @@ Query OK, 0 rows affected (0.01 sec)
 ## MySQL compatibility
 
 * In MySQL, with the exception of Group Replication with multiple primaries, it is not typical that a `COMMIT` statement could result in an error. By contrast, TiDB uses optimistic concurrency control and conflicts may result in `COMMIT` returning an error.
-* Be default, `UNIQUE` and `PRIMARY KEY` constraint checks are deffered until statement commit. This behavior can be changed by setting `tidb_constraint_check_in_place=TRUE`.
-* TiDB only has syntactic support for `CompletionTypeWithinTransaction`. Closing the connection or continuing to open a new transaction commit option is not supported.
+* Be default, `UNIQUE` and `PRIMARY KEY` constraint checks are defered until statement commit. This behavior can be changed by setting `tidb_constraint_check_in_place=TRUE`.
```
Comment (ran-huang) [inline sql-statements/sql-statement-commit.md:39]:
```suggestion
* Be default, `UNIQUE` and `PRIMARY KEY` constraint checks are deferred until statement commit. This behavior can be changed by setting `tidb_constraint_check_in_place=TRUE`.
```

**Pull request merged**